templates/vue/src/components/FileUpload.vue

```diff
@@ -113,6 +114,7 @@ export default {
         })
         .then(response => {
           this.$emit("input", response.data.data.url)
+          this.$emit("isUploading", false)
```

Comment:
I think you should wrap this in a `.finally(...)` instead, since in the event of an error the `then` block wouldn't be called.

Reply:
We don't catch any errors at all in this part either - is that a problem?

Reply:
Also, would it be sufficient to set isUploading to false before the reset hook during the finally?

Reply:
> We don't catch any errors at all in this part either - is that a problem?

The error part was fixed in another PR that's awaiting review at the moment. 

> Also, would it be sufficient to set isUploading to false before the reset hook during the finally?

Yes I think so.